le.sh

```diff
@@ -279,6 +279,9 @@ _initpath() {
   if [ -z "$CA_CERT_PATH" ] ; then
     CA_CERT_PATH="$WORKING_DIR/$domain/ca.cer"
   fi
+  if [ -z "$CERT_CHAIN_PATH" ] ; then
```

Comment:
What is the point of defining this variable as empty and then testing if it is still empty ?I don't like the way it's not configurable, hence the proposed alternative.